React/Base/RCTPlatformDisplayLink.h

```diff
@@ -9,8 +9,8 @@
 
 #include <TargetConditionals.h>
 
+#import <QuartzCore/CADisplayLink.h> // TODO GH#533, we need to explicitly pull in the framework to get the definition for CACurrentMediaTime()
```

Comment:
What does the TODO indicate in this case? From the perspective of a developer who might touch this code next, it's not clear to me what I should keep in mind or try to work towards.